DiscordBot/Bot.cs

```diff
@@ -27,16 +30,23 @@ public async Task RunAsync()
             var configJson = JsonConvert.DeserializeObject<BotConfigJSON>(json);
 
             var config = new DiscordConfiguration()
-            {
+            { 
                 Token = configJson.token,
                 TokenType = TokenType.Bot,
                 AutoReconnect = true,
                 MinimumLogLevel = Microsoft.Extensions.Logging.LogLevel.Debug
-                };
+               
+            };
         
             Client = new DiscordClient(config);
             Client.Ready += OnClientReady;
 
+            Client.UseInteractivity(new InteractivityConfiguration()
```

Comment:
This can be encapsulated in some well named method for example AddEmojiInteractivity() or something like this.